packages/core/src/App/Containers/Modals/app-modals.jsx

```diff
@@ -35,7 +35,14 @@ const CloseUKAccountModal = React.lazy(() =>
     import(/* webpackChunkName: "close-mx-mlt-account-modal" */ '../CloseUKAccountModal')
 );
 
+const WarningScamMessageModal = React.lazy(() =>
+    import(/* webpackChunkName: "warning-scam-message" */ '../WarningScamMessageModal')
+);
+
+const is_read = localStorage.getItem('readScamMessage') || false;
```

Comment:
You should move this to `client-store`, and change the name to something more meaningful e.g. `has_read_scam_message`